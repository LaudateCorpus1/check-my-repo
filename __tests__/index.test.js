const { printResults, createJsonFile, validateChangeLog } = require('../src/utils')
const results = require('./__fixtures__/results')
const resultsPassed = require('./__fixtures__/results-passed')
const { createdInstances } = require('@octokit/rest')

const fs = require('fs')
jest.mock('fs')
fs.promises = { mkdir: jest.fn(), writeFile: jest.fn() }

const repository = 'repository'
const organization = 'organization'

test('should return a sum of all repositories without fail', () => {})
