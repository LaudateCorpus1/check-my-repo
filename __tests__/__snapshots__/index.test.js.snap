// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should return a sum of all repositories without fail 1`] = `Array []`;
